app/index.php

```diff
@@ -1,9 +1,30 @@
 <?php
-session_start();
 require "../vendor/autoload.php";
+session_start();
 $conn = new \App\DbConnector();
 $store = new \App\Store($conn->getDb());
-$books = $store->getAllBooks();
+
+if (isset($_GET['search'])) {
+    $searchTerm = $_GET["search"];
+    $searchGetTerm = 'search=' . $_GET['search'] . "&";
+    $searchTitle = new \App\SearchTitle($conn->getDb());
+    $books = $searchTitle->performSearch($searchTerm);
+    $sortBooks = new \App\SortBooks($books);
+} else {
+    $searchTerm = "";
+    $searchGetTerm = "";
+    $books = $store->getAllBooks();
+    $filteredBooksPrices = $store->getAllBookPrices();
+    $sortBooks = new \App\SortBooks($filteredBooksPrices);
+}
+
+if ((!empty($_GET) && isset($_GET['min'])) && $_GET['min'] && $_GET['max']) {
+    $books = $store->getBooksWithinRange($_GET['min'], $_GET['max'], $searchTerm);
+}
+
+$bookPrices = $sortBooks->getBooksPriceAscending();
```

Comment:
what if this is empty? if someone searches for a term with no books, there will be no bookprices, so you cant create filters